roles/elasticsearch/templates/elasticsearch.yml.j2

```diff
@@ -0,0 +1,90 @@
+# {{ ansible_managed }}
+#
+# ======================== Elasticsearch Configuration =========================
+#
+# NOTE: Elasticsearch comes with reasonable defaults for most settings.
+#       Before you set out to tweak and tune the configuration, make sure you
+#       understand what are you trying to accomplish and the consequences.
+#
+# The primary way of configuring a node is via this file. This template lists
+# the most important settings you may want to configure for a production cluster.
+#
+# Please consult the documentation for further information on configuration options:
+# https://www.elastic.co/guide/en/elasticsearch/reference/index.html
+#
+# ---------------------------------- Cluster -----------------------------------
+#
+# Use a descriptive name for your cluster:
+#
+#cluster.name: my-application
```

Comment:
I've added this one in because it makes sense. Very little about the geerlingguy roles was actually configurable, so for the initial i was going for the minimum options and then add things as we need it.